testutil/docker.go

```diff
@@ -19,7 +19,10 @@ func IsDockerCase(testName string) bool {
 // SkipDockerIfUnsupported skips the test if the platform is not Linux or the
 // SKIP_DOCKER_TESTS environment variable is explicitly set.
 func SkipDockerIfUnsupported(t *testing.T) {
-	skip, _ := strconv.ParseBool(os.Getenv("SKIP_DOCKER_TESTS"))
+	skip, err := strconv.ParseBool(os.Getenv("SKIP_DOCKER_TESTS"))
+	if err == nil && !skip {
```

Comment:
Wouldn't this get taken care of on line 26?

Reply:
This handles the case in which you set `SKIP_DOCKER_TEST=false` on a non-Linux machine (e.g. you're on MacOS with `SKIP_DOCKER_TESTS=false` for local testing). L26 this handles the case in which `SKIP_DOCKER_TESTS=true` or you're on a non-Linux machine, which by default skips Docker tests.